docs/src/main/sphinx/release/release-469.md

```diff
@@ -0,0 +1,186 @@
+# Release 469 (21 Jan 2025)
+
+## General
+
+* Add support for the `FIRST`, `AFTER`, and `LAST` clauses to `ALTER TABLE ...
+  ADD COLUMN`. ({issue}`20091`)
+* Add the geospatial {func}`ST_GeomFromKML` function. ({issue}`24297`)
+* Allow deactivating the spooling protocol with the `spooling_protocol_enabled`
```

Comment:
Should we collapse these and not add any of the property names since they are added to the docs in #24761 @wendigo ?

Reply:
yeah, makes sense

Reply:
Done